File: Teledyne DALSA/Sapera/Demos/NET/FlatFieldDemo/VB/FlatFieldDemo.vb
Public Class FlatFieldDemo
   'The main entry point for the application.
   <STAThread()> _
   Shared Sub Main()
      Application.EnableVisualStyles()
      Application.SetCompatibleTextRenderingDefault(False)
      Dim FF As FlatFieldDemoDlg = New FlatFieldDemoDlg()
      If Not FF.IsDisposed Then
         Application.Run(FF)
      End If
   End Sub
End Class

File: Teledyne DALSA/Sapera/Demos/NET/FlatFieldDemo/CSharp/FlatFieldDemo.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;

namespace DALSA.SaperaLT.Demos.NET.CSharp.FlatFieldDemo
{
    static class FlatFieldDemo
    {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main()
        {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            FlatFieldDemoDlg dlg = new FlatFieldDemoDlg();
            if (!dlg.IsDisposed)
               Application.Run(dlg);
        }
    }
}